src/ontology/components/relation.owl

```diff
@@ -74,6 +81,20 @@
     
 
 
+    <!-- http://purl.obolibrary.org/obo/VBO_0300020 -->
+
+    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/VBO_0300020">
```

Comment:
Are we totally positive that `VBO:0300020` should be an Object property? Remember that it gets _always_ inherited down the hierarchy.

Reply:
I cannot think of an instance when this property should not be inherited down the hierarchy, but I better check with @sabrinatoro.

Reply:
I agree with @katiermullen, any children of a term with this relation should always inherit this relation. 
This relation is to be used to define a term. So all children should also have it (note that I doubt that we would have a lot of children for these terms)

Reply:
Great! Closing this thread.